src/execute_lua_script.cc

```diff
@@ -79,4 +83,14 @@ execute_lua_script(sol::state& lua, sol::string_view script)
     }
 }
 
+gul14::expected<sol::load_result, std::string>
+load_lua_script(sol::state& lua, sol::string_view script)
+{
+    sol::load_result result = lua.load(script, anchor);
+    if (result.valid())
```

Comment:
No, that would not work because the expressions on the two sides of the `:` have different types. We would have to put them into a full `expected<..., ...>{}` constructor.